root/report/ArtistCreditsWithDubiousTrailingPhrases.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import ArtistCreditList from './components/ArtistCreditList';
+import ReportLayout from './components/ReportLayout';
+import type {ReportArtistCreditT, ReportDataT} from './types';
+
+const ArtistCreditsWithDubiousTrailingPhrases = ({
+  canBeFiltered,
+  filtered,
+  generated,
+  items,
+  pager,
+}: ReportDataT<ReportArtistCreditT>): React.Element<typeof ReportLayout> => (
+  <ReportLayout
+    canBeFiltered={canBeFiltered}
+    description={l(
+      `This report lists artist credits that have a trailing join phrase
+       that looks like it might have been left behind in error, such as
+       a trailing comma or “feat.”.`,
+    )}
+    entityType="artist_credit"
+    filtered={filtered}
+    generated={generated}
+    title={l('Artist credits with dubious trailing phrases')}
```

Comment:
Maybe include "join" here for clarity? Understandable if you think it makes the package name too long, but it should be fine in the title.
```suggestion
    title={l('Artist credits with dubious trailing join phrases')}
```